docs/Installation.md

```diff
@@ -36,3 +36,12 @@ baseline: spec
 		do: [ self setUpDependencies: spec.
 			spec package: 'My-Package' with: [ spec requires: #('Buoy') ] ]
 ```
+
+## Provided groups
+
+- `Deployment` will load all the packages needed in a deployed application
```

Comment:
`Deployment` will load all the packages needed in a deployed application. Aimed to be used when setting Buoy as a dependency.

What do you think? I'm starting to believe that we should rename this group in all of the @ba-st projects.

Reply:
It's the group you want to depend on for the deployment of your app, but not the only dependency you can have on Buoy.. for the development of your app I would include the Tools group for example. I'm happy with the name.

Reply:
Ok, good point. Thanks!